docs/basics/getting-started.md

```diff
@@ -1,210 +1,159 @@
-  -o . \ # any directory, cross-seed will put its output torrents here
-  --torznab https://localhost/prowlarr/1/api?apikey=12345 # any valid torznab link, separated by spaces
+docker run \
+-v /path/to/config:/config \
+ghcr.io/cross-seed/cross-seed gen-config
 ```
 
-If you were lucky, you've already got your first cross-seed in your current
-directory!
-
-:::info
-`cross-seed`, by default, is particular about the contents of torrents you feed
-it. It will ignore torrents with non-video files in them, and it will ignore
-torrents that look like single episodes.
-
-Use the command line flags to override any defaults:
-`--include-episodes --include-non-videos`
-
-These command line arguments are always configurable in the config.js file (generated with gen-config).
-You won't have to specify them if they are set unless you wish to override the settings temporarily.
+This will create a `config.js` file inside the `/config` directory you
+specified. You’ll need to open this file and edit it as described below.
 
-Below we will cover the configuration process.
-:::
+---
 
-## Scaling Up
+### 3. Edit the Config File
+
+Once you’ve created your config file, you’ll need to fill in the necessary
+fields to connect to your torrent client and indexer. At a minimum, you should
+configure the following:
+
+-   **`torznab`**: URLs of your Torznab indexers (from Prowlarr or Jackett).
+-   **`torrentDir`**: The directory where your .torrent files are located (from
+    your torrent client).
+-   **`outputDir`**: Create a new directory somewhere to house the torrent files `cross-seed` outputs.
+-   **Connection to your torrent client**: Depending on your client, you'll need
+    to specify one of the following:
+    -   `rtorrentRpcUrl`: For rTorrent users. Often looks like `http://user:pass@localhost:8080/RPC2`
+    -   `qbittorrentUrl`: For qBittorrent users. Often looks like `http://user:pass@localhost:8080`
+    -   `transmissionRpcUrl`: For Transmission users. Often looks like `http://user:pass@localhost:9091/transmission/rpc`
+    -   `delugeRpcUrl`: For Deluge users. Often looks like `http://:pass@localhost:8112/json` (the colon before pass is intentional)
+
+Here’s an example of what your `config.js` might look like:
+
+```js
+module.exports = {
+	torznab: ["http://localhost:9696/1/api?apikey=123456"],
+	torrentDir: "/path/to/torrent_dir",
+    outputDir: "/path/to/output_dir",
+	qbittorrentUrl: "http://username:password@localhost:8080",
+	... several other settings you can read later ...
+};
+```
 
-Searching like the above is nice, but it's pretty manual and not very useful in
-practice. In this section, we'll set up a configuration file. A configuration
-file is not necessary to use cross-seed, but it makes things a lot easier.
+For more details on configuring the connection to your torrent client, refer to
+the [reference guide on torrent client configuration](../tutorials/injection.md).
 
```

Comment:
````suggestion
};
```

For details on configuring the connection to your torrent client, refer to
the [reference guide on torrent client configuration](../tutorials/injection.md).

You can find more information on other options on the 
[options page](./options.md#options-used-in-cross-seed-daemon) once you 
have finished configuring the basics.

````

Something like this might be a bit more guiding. Potentially with a tip/info box, but I'm not sure that's really necessary.

Reply:
added this to next steps at the bottom in https://github.com/cross-seed/cross-seed.org/pull/58/commits/3b6f8113b5e04d64c78dd54d5d1e4ec451b83e71